sdk/highlight-next/src/util/with-highlight-edge.ts

```diff
@@ -19,7 +19,7 @@ export function Highlight(env: HighlightEnv) {
 		) {
 			H.initEdge(request, env, event)
 			const headers: IncomingHttpHeaders = {}
-			request.headers.forEach((k, v) => (headers[k] = v))
+			request.headers.forEach((v, k) => (headers[k] = v))
```

Comment:
This was baffling at first, but MDN explains the flipped args:

https://developer.mozilla.org/en-US/docs/Web/API/Headers/forEach